test/is_clamped_test.cpp

```diff
@@ -0,0 +1,224 @@
+﻿//  (C) Copyright TODO
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <boost/algorithm/clamp.hpp>
+#include <boost/algorithm/is_clamped.hpp>
+#include <cmath>
+#include <cstdint>
+#ifdef __cpp_impl_three_way_comparison
+#include <compare>
```

Comment:
note for myself:
due to magic of C++ this is probably not a proper check, compare header can be unimplemented even if language support is there, so also __has_include is needed.

Reply:
done